README.md

```diff
@@ -547,6 +547,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [amp](https://amp.rs) - Inspired by Vi/Vim.
 * [emacs-ng](https://github.com/emacs-ng/emacs-ng) - Complementing the C codebase with rust code to introduce new features.
 * [gchp/iota](https://github.com/gchp/iota) - A simple text editor
+* [gladius](https://codeberg.org/njskalski/bernardo/src/branch/master/docs/beta_release_notes/beta_1_release.md) - A command line, keyboard only, multicursor code editor with LSP and sane defaults.
```

Comment:
Couple of problems here:
* That's a link to release notes for a particular version of the software, and we need a link to the actual project
* Our usual metrics for inclusion are 50 stars or 2000 downloads of crates. I'm only seeing 12 stars on this repository, no crate immediately visible, and no other immediately obvious public metrics for the popularity of this project. Can you point me at anything in particular I've missed?